src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -102,8 +107,9 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
   const [title, setItemTitle] = React.useState<string>('');
   const [objectType, setObjectType] = React.useState<string>('');
   const [arrayType, setArrayType] = React.useState<string>('');
-  const [objectKind, setObjectKind] = React.useState<'type' | 'reference' | 'group'>('type');
+  const [objectKind, setObjectKind] = React.useState<'type' | 'reference' | 'combination'>('type');
```

Comment:
Isn't this type defined in types?
Is the idea that there is a hierarchy of `type` > `kind`, and if so why not `type` > `subType`?

Reply:
No, the field type and the object kind are two different concepts. 
Some sort of hierarchy here for our components might be possible though, things feel a little mixed together to make it all work, but we should take it step by step.